frontend/AnimeCore/src/lib/components/pages/anime_info/AnimeMainDetails.svelte

```diff
@@ -0,0 +1,155 @@
+	import Edit from '$icons/Edit.svelte';
+</script>
+
+<div class="grid grid-cols-12 items-start">
+	<div class="col-span-10 grid grid-cols-12 items-end xl:gap-10">
+		<div class="col-span-7 flex items-end gap-10">
+			<img class="xl:w-60 2xl:w-80 xl:rounded-xl 2xl:rounded-3xl" src={anime.cardBackgroundImage} alt={anime.titles.eng} />
+			<div>
+				<h2 class="font-bold 2xl:!text-5xl 3xl:!text-6xl">{anime.titles.eng}</h2>
+
+				<p class="pt-3 3xl:mt-5 flex items-center gap-2 text-surface-100">
+					<span class="xl:text-xs 2xl:text-sm 3xl:!text-[16px] font-medium after:content-['▪'] after:ml-1">{anime.titles.japanese}</span>
+					<span class="xl:text-xs 2xl:text-sm 3xl:!text-[16px] font-medium after:content-['▪'] after:ml-1">{anime.titles.eng}</span>
+					<span class="xl:text-xs 2xl:text-sm 3xl:!text-[16px] font-medium">{anime.titles.others}</span>
+				</p>
+
+				<p class="pt-4 flex flex-wrap items-center gap-2 pr-5">
+					<span class="xl:text-xs 2xl:text-sm 3xl:!text-[16px] 2xl:!leading-none font-semibold after:content-['▪'] after:ml-1">
+						{anime.type}
+					</span>
+					<span class="xl:text-xs 2xl:text-sm 3xl:!text-[16px] 2xl:!leading-none font-semibold after:content-['▪'] after:ml-1">
+						{anime.episodes} eps
+					</span>
+					<span class="xl:text-xs 2xl:text-sm 3xl:!text-[16px] 2xl:!leading-none font-semibold after:content-['▪'] after:ml-1">
+						{anime.status}</span>
+					<span class="xl:text-xs 2xl:text-sm 3xl:!text-[16px] 2xl:!leading-none font-semibold after:content-['▪'] after:ml-1">
+						{anime.premiered}
+					</span>
+					<span class="xl:text-xs 2xl:text-sm 3xl:!text-[16px] 2xl:!leading-none font-semibold">{anime.studio}</span>
+				</p>
+
+				<div class="xl:mt-5 2xl:mt-7 3xl:mt-12 flex items-center xl:gap-3 2xl:gap-5">
+					<button
+						type="button"
+						class="btn 2xl:btn-md 3xl:btn-lg xl:h-16 2xl:h-20 justify-start xl:rounded-lg 2xl:rounded-xl bg-primary-500 xl:text-xs 2xl:text-xl font-bold text-white"
+					>
+						<PlayCircle width="30" height="30" color="white" />
+						<div class="flex flex-col items-start">
+							<span class="leading-5">Watch</span>
+							<span class="xl:text-[0.6rem] 2xl:text-lg font-normal leading-none opacity-70">Ep 01</span>
+						</div>
+					</button>
+
+					<button
+						type="button"
+						class="btn xl:h-16 2xl:h-20 2xl:w-20 3xl:w-24 flex-col gap-1 xl:rounded-lg 2xl:rounded-xl bg-secondary-100 xl:text-xs 2xl:text-base 3xl:text-xl xl:font-bold 2xl:font-semibold text-surface-500"
+					>
+						<Read width="22" height="22" color="bg-surface-500" />
+						Read
+					</button>
+
+					<button
+						type="button"
+						class="btn xl:h-16 2xl:h-20 2xl:w-20 3xl:w-24 flex-col gap-1 xl:rounded-lg 2xl:rounded-xl bg-secondary-100 xl:text-xs 2xl:text-base 3xl:text-xl xl:font-bold 2xl:font-semibold text-surface-500"
+					>
+						<Listen width="22" height="22" color="bg-surface-500" />
+						Listen
+					</button>
```

Comment:
Buttons look 4px smaller than it needs to be

Reply:
Ok will do that